src/poliastro/tests/test_frames.py

```diff
@@ -124,8 +124,8 @@ def test_round_trip_from_GeocentricSolarEcliptic_gives_same_results():
 def test_GeocentricSolarEcliptic_against_data():
     gcrs = GCRS(ra="02h31m49.09s", dec="+89d15m50.8s", distance=200 * u.km)
     gse = gcrs.transform_to(GeocentricSolarEcliptic(obstime=Time("J2000")))
```

Comment:
Here is the problem: we should have used `poliastro.constants.J2000`, which explicitly specifies the scale. My fault for not spotting it when I reviewed the code.

Reply:
Great! Thank you @Juanlu001, I saw in the changelog that the default scale was changed but thought it was because of the `float64`. Solving immediately :+1: